### PR TITLE
Add reinvite button in user edit dialog

### PR DESCRIPTION
**Pull request by hrvojevu**
This PR extends already implemented `reinvite` functionality by adding `Reinvite` button in user edit dialog

### CODE REVIEW

File: client/components/system-settings/UserManagement/UserDialog.vue [hunk 1]
```diff
@@ -2,6 +2,17 @@
   <tailor-dialog v-model="show" header-icon="mdi-account">
     <template v-slot:header>{{ userData ? 'Edit' : 'Create' }} User</template>
     <template v-slot:body>
+      <div v-if="userData" class="text-left">
+        <v-btn
+          @click="reinvite"
+          :loading="isLoading"
+          :disabled="isLoading"
+          outlined
+          color="primary"
```
Comment (underscope) [inline client/components/system-settings/UserManagement/UserDialog.vue:11]: Attr order

File: client/components/system-settings/UserManagement/UserDialog.vue [hunk 2]
```diff
@@ -2,6 +2,17 @@
   <tailor-dialog v-model="show" header-icon="mdi-account">
     <template v-slot:header>{{ userData ? 'Edit' : 'Create' }} User</template>
     <template v-slot:body>
+      <div v-if="userData" class="text-left">
+        <v-btn
+          @click="reinvite"
+          :loading="isLoading"
+          :disabled="isLoading"
```
Comment (underscope) [inline client/components/system-settings/UserManagement/UserDialog.vue:9]: We should probably rename this flag ?

**Comment by kjuej** [comment]
@hrvojevu @underscope Would it make more sense to place a reinvite button in the actions column (mdi-refresh or something)?

**Comment by hrvojevu** [comment]
@kjuej That is fine, but in that case we would need to add confirmation dialog for reinvite. Additionally, I'm not sure about the `mdi-refresh` icon since it doesn't imply reinvite and in that case we would have to add tooltips.

**Comment by underscope** [comment]
@kjuej Since reinvite should be rare action lets keep it this way for now (don't want to pollute the user list). In terms of verbiage, we should consider moving from 'edit' to 'manage' user terminology.

**Pull request merged**